service/core/serverObj/v2ray.go

```diff
@@ -228,21 +234,35 @@ func (v *V2Ray) Configuration(info PriorInfo) (c Configuration, err error) {
 				},
 			}
 		case "vless":
+			security := v.Security
+			if security == "" {
+				security = "auto"
+			}
+			flow := v.Flow
+			if flow == "" {
+				flow = "xtls-rprx-direct"
+			}
 			core.Settings.Vnext = []coreObj.Vnext{
 				{
 					Address: v.Add,
 					Port:    port,
 					Users: []coreObj.User{
 						{
-							ID:         id,
 							Encryption: "none",
+							Flow:       flow,
+							ID:         id,
+							Level:      8,
+							Security:   security,
 						},
 					},
 				},
 			}
-		}
-		core.StreamSettings = &coreObj.StreamSettings{
-			Network: v.Net,
+			tcpSetting := coreObj.TCPSettings{
```

Comment:
Do we really need TCPSettings?

Reply:
Good point.

Fixed, and now we only setting TCP headers to none when we used TCP network

Reply:
May I ask why TCPSetting is needed and can it be omitted?
If it cannot, please move it below (line 299)

Reply:
Bcz at line 300 (if strings.ToLower(v.Type) == "http" {) we check if type is http we set tcp but by default it's could be set Header none (Vless Only!) to prevent any unsolicited requests on server.